### PR TITLE
add bert train example on CI

### LINKED ISSUE #733: add bert training example
Add a bert training example with TorchBlade accelerator in the codebase and check the loss on CI. This task needs to check the disc backend with LTC tracing modular at least, and Dynamo in the future.

cc @tanyokwok

### PR DESCRIPTION
**Pull request by Yancey1989**
fixed #733

### CODE REVIEW

File: pytorch_blade/examples/check_bert_loss.py [hunk 1]
```diff
@@ -0,0 +1,304 @@
+# Copyright 2022 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
```
[the remark this answers is not in the document]
Comment (Yancey1989) [reply pytorch_blade/examples/check_bert_loss.py:2]: We don't have an inference example yet, maybe don't need the second-level folder? But I can rename the script to `check_bert_train_loss.py` to be more meaningful.

[the remark this answers is not in the document]
Comment (tanyokwok) [reply pytorch_blade/examples/check_bert_loss.py:2]: If you have concerns about it. I think the script can be moved to `examples/PyTorch/Training` in the root directory of BladeDISC.

[the remark this answers is not in the document]
Comment (Yancey1989) [reply pytorch_blade/examples/check_bert_loss.py:2]: That's a good idea~

[the remark this answers is not in the document]
Comment (Yancey1989) [reply pytorch_blade/examples/check_bert_loss.py]: done

**Review by tanyokwok — APPROVED** [review]
LGTM
